model_cards/jimregan/BERTreach/README.md

```diff
@@ -0,0 +1,21 @@
+---
+language: ga
```

Comment:
Did you get a chance to test the Opus-MT translation models for Irish? https://huggingface.co/models?filter=ga

Are they good?

Reply:
I haven't looked. I worked with MT for several years, did a master's on it and burned myself out completely writing the dissertation, so I wasn't really inclined to check. 
I get 'Can't load config' on both models from the website; the English->Irish model seems good enough for the sort of short sentences I can confidently rate.

Reply:
Understood, thanks!

cc @sshleifer and @mfuntowicz for the inference API issue